services/cii-best-practices/cii-best-practices.tester.js

```diff
@@ -0,0 +1,252 @@
+'use strict'
+
+const Joi = require('joi')
+const t = require('../create-service-tester')()
+module.exports = t
+
+t.create('live: level known project')
+  .get(`/level/1.json`)
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: 'cii',
+      value: Joi.string(),
+    })
+  )
+
+t.create('live: percentage known project')
+  .get(`/percentage/29.json`)
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: 'cii',
+      value: Joi.string(),
+    })
+  )
+
+t.create('live: default known project')
+  .get(`/default/33.json`)
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: 'cii',
+      value: Joi.string(),
+    })
+  )
+
+t.create('live: unknown project')
+  .get(`/level/abc.json`)
+  .expectJSON({ name: 'cii', value: 'project not found' })
+
+t.create('level: unknown project')
```

Comment:
I think the `live: unknown project` test is enough as far as unknown projects are concerned. The bit of code we're testing is the `404: 'project not found'`, having three additional mocked tests will not improve our coverage nor give us more confidence that the badge is wired up correctly.

Reply:
Yes, my suggestion was initially to remove all the project not found tests apart from the live one. You're right by saying that we might need additional ones in the future if the implementation changes, however in the meantime they (slightly) slow down our test suite and add code to maintain. ^^